test/karma.conf.js

```diff
@@ -57,13 +64,13 @@ module.exports = function(config) {
     },
 
     // web server port
-    port: 9443,
+    port: 4443,
```

Comment:
port change needed?

Reply:
I think someone else changed the port in another PR, but it was 8443 before, which conflicted with the api server if it was running when you try to run tests.